storage/bucket.go

```diff
@@ -131,6 +131,7 @@ func (b *BucketHandle) DefaultObjectACL() *ACLHandle {
 
 // Object returns an ObjectHandle, which provides operations on the named object.
 // This call does not perform any network operations.
+// The object is only created in the bucket (if it does not already exist) once you write to the object.
```

Comment:
I think this is basically implied by the previous line? Maybe change it to this:

```
// This call does not perform any network operations such as fetching the object or verifying its existence.
```

How does that sound?